libraries/src/Menu/AbstractMenu.php

```diff
@@ -364,4 +389,19 @@ public function load()
 	{
 		return array();
 	}
+
+	/**
+	 * Internal getter for the user. Returns the injected
+	 * one if any, or the current one if none.
+	 *
+	 * @return User|\JUser
```

Comment:
Since you import the namespace you should use only "User"
```suggestion
	 * @return User
```

Reply:
OK

Reply:
I thought the problem is because the object is created but not used so it doesn't call getUser() but on the other side the object should always use the current (or set) user so you are right.